ikenna-portfolio/portfolio/src/main/webapp/index.html

```diff
@@ -119,15 +117,15 @@ <h4 class="margin">We promise to take your posts seriously >:)</h4>
         <div class="comment-wrapper panel-info panel-body">
           <textarea name="name-box" class="form-control" placeholder="Name (Not Required)" rows="1"></textarea><br>
           <textarea name="text-box" class="form-control" placeholder="Questions?, Comments., Concerns!" rows="3"></textarea><br>
-          <input type="submit" class="btn btn-info pull-right">
+          <input type="submit" class="btn btn-info float-right">
```

Comment:
Since you can only leave a comment when logged in now, can you disable the submit button when the user is not logged in? 

Also, you might already have this in a future change, but it doesn't seem necessary to have the name field since there is the nickname field in User.

Reply:
done; will add a checkbox for anonymous posts and a way to set nickname in future.

Reply:
added anonymous